requirements.txt

```diff
@@ -1,5 +1,7 @@
 # scc-hypervisor-collector direct dependencies
+importlib-metadata
```

Comment:
This will add a dependency on the `python3-importlib-metadata` package in the spec file...
In the spec file, find the `%package common` section of the file, and add a line
```
Requires:       %{python_module importlib-metadata}
```
above the similar line that specifies the PyYAML dependency.

Reply:
Oh and near the top of the file, you will need to add a similar `BuildRequires: ...` entry...